src/components/game/city/city.actions.ts

```diff
@@ -5,12 +5,14 @@ import { Player } from '../game.types'
 
 export enum ActionTypes {
     ADD_CITY = '[city] add city',
-    CREATE_CITY = '[city] create city'
+    CREATE_CITY = '[city] create city',
+    BUILD = '[city] build'
 }
 
 export const actions = {
     addCity: (city: City) => createAction(ActionTypes.ADD_CITY, city),
-    createCity: (position: GridPosition, owner: Player) => createAction(ActionTypes.CREATE_CITY, { owner, position })
+    createCity: (position: GridPosition, owner: Player) => createAction(ActionTypes.CREATE_CITY, { owner, position }),
+    build: (city: City, buildKey: string) => createAction(ActionTypes.BUILD, { city, buildKey })
```

Comment:
what is `buildKey`?

Reply:
should it be `buildingId`? or `itemId` since it can also be units or other items